service/movie-information/src/main/java/com/ttbmp/cinehub/service/movieinformation/mock/MockMovieApiService.java

```diff
@@ -0,0 +1,40 @@
+package com.ttbmp.cinehub.service.movieinformation.mock;
+
+
+import com.ttbmp.cinehub.core.dto.MovieDto;
+import com.ttbmp.cinehub.core.service.movie.MovieApiService;
+
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Palmieri Ivan
+ */
+public class MockMovieApiService implements MovieApiService {
```

Comment:
The operations defined in this interface return void instead of return what you would expect from their name

Reply:
The mock class should not connect to an external service to work, it should just return mocked objects.

Reply:
Now the mock has only one method that repeats a movie list
As regards movie api, the methods not necessarily interested in the user, such as the connection or the creation of Gson objects, have become private, the MovieApiService interface has also been modified, which for now has only one function to return a list of movie